pkg/psmdb/service.go

```diff
@@ -42,9 +42,10 @@ func Service(cr *api.PerconaServerMongoDB, replset *api.ReplsetSpec) *corev1.Ser
 		Spec: corev1.ServiceSpec{
 			Ports: []corev1.ServicePort{
 				{
-					Name:       mongodPortName,
-					Port:       api.DefaultMongodPort,
-					TargetPort: intstr.FromInt(int(api.DefaultMongodPort)),
+					Name:        mongodPortName,
+					Port:        api.DefaultMongodPort,
+					TargetPort:  intstr.FromInt(int(api.DefaultMongodPort)),
+					AppProtocol: &appProtocol,
```

Comment:
@egegunes @sorenmat  should we do the same for `service per pod`?
how to enable:
```
spec:
  replsets:
  ..
    expose:
      enabled: true
      exposeType: ClusterIP
```
and what about service for mongos

Reply:
@sorenmat, any update from your end? Do you have time for it?

Reply:
@sorenmat we need to add this field only for the latest version (1.16.0). please add a version check like https://github.com/percona/percona-server-mongodb-operator/blob/main/pkg/psmdb/service.go#L55-L65